pkg/apis/config/types.go

```diff
@@ -35,6 +35,9 @@ type ControllerConfiguration struct {
 	// Auth is configuration for metal stack specific user authentication in the cluster.
 	Auth Auth
 
+	// Audit is configuration for auditlogging.
+	Audit Audit
```

Comment:
Same here. A little more specific on what this is meant for in order to try not to confuse adopters more than necessary.

Reply:
Expanded the comment to better explain what it is for.